.github/actions/composite/setupNode/action.yml

```diff
@@ -9,7 +9,15 @@ runs:
         node-version-file: '.nvmrc'
         cache: npm
 
+    - id: cache-node-modules
+      if: github.run_attempt == '1'
+      uses: actions/cache@v3
+      with:
+        path: node_modules
+        key: ${{ runner.os }}-node-modules-${{ hashFiles('package-lock.json') }}
+
     - name: Install node packages
+      if: github.run_attempt != '1' || steps.cache-node-modules.outputs.cache-hit != 'true'
```

Comment:
Note: I tried using `fromJSON` here, but it doesn't work because `steps.cache-node-modules.outputs.cache-hit` is either `true` or `null`, not `true` or `false` as the documentation suggests.

Reply:
👍 I see this matches the example in the docs [here](https://docs.github.com/en/actions/using-workflows/caching-dependencies-to-speed-up-workflows#example-using-the-cache-action)